installer/PowerToysSetup/Product.wxs

```diff
@@ -39,6 +39,8 @@
       <ComponentGroupRef Id="CoreComponents" />
       <ComponentGroupRef Id="ResourcesComponents" />
     </Feature>
+    <SetProperty Id="ARPINSTALLLOCATION" Value="[INSTALLFOLDER]" After="CostFinalize" />
+
     <Property Id="WIXUI_INSTALLDIR" Value="INSTALLFOLDER" />
```

Comment:
Any idea why in this case the square brackets are not needed?

Reply:
No idea, perhaps because its evaluation has to be deferred? I've seen it in the official wix docs https://wixtoolset.org//documentation/manual/v3/wixui/dialog_reference/wixui_installdir.html